api/envoy/api/v2/route/route.proto

```diff
@@ -604,6 +605,13 @@ message RouteAction {
       bool source_ip = 1;
     }
 
+    message QueryParameter {
+      // The name of the URL query parameter that will be used to obtain the hash
+      // key. If the parameter is not present, no hash will be produced. Query
+      // parameter names are case-sensitive.
+      string parameter_name = 1 [(validate.rules).string = {min_bytes: 1}];
```

Comment:
From a @envoyproxy/api-shepherds  perspective, I think `name` is shorter and unambiguous here, so suggest switching to that.